app/views/pages/get-started/build-your-first-app/building-the-user-interface.liquid

```diff
@@ -0,0 +1,195 @@
+
+There will be many other tags and uses of Liquid in the future. This particular one is later automatically replaced by the actual page content that we will define in other `.liquid` files. So the content of other pages will be injected in place for {% raw %}`{{ content_for_layout }}`{% endraw %}. This makes the layout file a universal page structure for all of our content.
+
+{% include 'alert/tip', content: 'Remember that you need to [synchronise](/get-started/working-with-the-code-and-files/#upload-code-changes-automatically-when-you-save-the-file) or deploy your changes to the instance to see the results.' %}
+
+We can see this in action by creating another file – this one with the content that will be shown on the index page of our application.
+
+All of the pages (or _subpages_) of our web application needs to be placed in a separate folder: `app/views/pages/`. Start by creating the index file with a simple text:
+
+#### app/views/pages/index.liquid
+
+```html
+<h2>Hello world!</h2>
+```
+
+Now, when you go to your instance URL, you should see the application title that we defined in `application.html.liquid` as well as the content of `index.liquid`. Check out the source, it’s all there!
+
+
+## Add new page to your application
+
+Let’s say you would like to add an About me page to your application, that will show some personal information. We would like this page to be available under `/about` URL.
+
+On platformOS this is a super straight-forward operation. When you add a new file called `about.liquid` and save it in `app/view/pages/` folder, it will automatically be visible under /about URL on your instance. Basically – the file name becomes the URL.
+
+Ok, so let us actually do this. As described above – create a new file called `about.liquid` in `app/view/pages/` directory:
+
+#### app/views/pages/contact.liquid
+
+```html
+<h2>About me</h2>
+<p>
+	Hello! My name is [your name] and this is my first app built on platformOS
+</p>
+```
+
+Now, when you go to `<your instance url>/about`, you should see the newly created page.
+
+Later on, you might find a need to create more complex URLs that are not entirely based on the file names, but on some query parameters for example. This is all totally possible and it’s described in the Routing section of this documentation.
+
+
+## A To Do App: Necessary Interface Elements
+
+<img loading="lazy" src="{{ 'images/get-started/todo/ui.jpg' | asset_url }}" alt="The layout we are implementing for our To do app">
+
+Back to our _To Do_ application – on the index page we would need: a list of items (with a button to mark an item as done), an information shown when there are no items, a simple form for adding a new item to the list and – finally – the link to the _About me_ page:
+
+#### app/views/pages/index.liquid
+
+```html
+<!-- list of items -->
+<ul>
+	<li>
+		First item
+		<form action="" method="post">
+			<button type="submit">Mark as done</button>
```

Comment:
it's a common pitfall that people forget to include CSRF token and then wonder why the session is invalidated. I'm wondering if it's worth making people used to it and having in example with a promise to explain it later?

Reply:
Let me think about it and I'll get back to this after writing the password protection tutorial.